tasks/install_shell.sh

```diff
@@ -407,8 +407,9 @@ install_file() {
       ;;
     "deb")
       info "installing puppetlabs apt repo with dpkg..."
-      dpkg -i "$2"
+      dpkg -i --force-conflicts "$2"
```

Comment:
This is because the rpm writes to the same repo.d file? Since we know the contents, this seems fairly safe. It forces only one Puppet collection at a time.